package.json

```diff
@@ -38,8 +38,10 @@
     "@codemirror/state": "^6.1.4",
     "@codemirror/view": "^6.6.0",
     "@types/d3": "^7.4.0",
+    "@types/d3-zoom": "^3.0.2",
     "d3": "^7.6.1",
     "d3-svg-to-png": "^0.2.1",
+    "d3-zoom": "^3.0.0",
```

Comment:
Did you use these dependency? What it the purpose of adding this?

Reply:
It's already used in mindmap-view.ts. It just wasn't in the dependencies.

Reply:
I removed the dependency on d3-zoom but kept the types